api/v1alpha1/workspace_validation.go

```diff
@@ -209,10 +209,6 @@ func (r *DataSource) validateCreate() (errs *apis.FieldError) {
 	if len(r.URLs) > 0 {
 		sourcesSpecified++
 	}
-	if r.Volume != nil {
-		errs = errs.Also(apis.ErrInvalidValue("Volume support is not implemented yet", "Volume"))
-		sourcesSpecified++
-	}
```

Comment:
@Fei-Guo We want this API to be changed in alpha version as well I assume? Just confirming